From 2f5aafc8ee22463c25286cab8bb9533f9f7d9bab Mon Sep 17 00:00:00 2001
From: Wang ShaoBo <bobo.shaobowang@huawei.com>
Date: Thu, 30 Dec 2021 16:26:48 +0800
Subject: [PATCH] configs: Enable osnoise and timerlat
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 2f5aafc8ee22463c25286cab8bb9533f9f7d9bab


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4G64B
CVE: NA

-------------------------------------------------

Enable CONFIG_OSNOISE_TRACER and CONFIG_TIMERLAT_TRACER by default y.

Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 83691c2ad677..63436eaa1ada 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -7149,3 +7149,5 @@ CONFIG_NET_VENDOR_RAMAXEL=y
 CONFIG_SPNIC=m
 CONFIG_SPFC=m
 CONFIG_GPIO_HISI=y
+CONFIG_OSNOISE_TRACER=y
+CONFIG_TIMERLAT_TRACER=y
-- 
2.26.2

